### PR TITLE
[DOCFIX] Add alluxio presto sandbox AMI docs

### PR DESCRIPTION
**Pull request by Xenorith**
mirroring content from https://www.alluxio.io/products/aws/alluxio-presto-sandbox-aws/

### CODE REVIEW

File: docs/en/cloud/Alluxio-Presto-AMI.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+
+## Launch an EC2 instance with Presto + Alluxio stack
+{% accordion launch %}
+  {% collapsible Open the web console %}
+Open a web browser and navigate to the [EC2 console](https://console.aws.amazon.com/ec2), logging in with your credentials when prompted.
+Note the AWS region in the upper right corner.
+
+![presto_sandbox_ec2_region]({{ '/img/presto_sandbox_ec2_region.png' | relativize_url }})
+
+Click on the region to open a dropdown of available regions.
```
Comment (LuQQiu) [inline docs/en/cloud/Alluxio-Presto-AMI.md]: region -> `region`?

File: docs/en/cloud/Alluxio-Presto-AMI.md [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+In the left sidebar, there are 4 categories: Quick Start, My AMIs, AWS Marketplace, and Community AMIs.
+Click on `AWS Marketplace` and search for the Alluxio-Presto sandbox AMI by typing `alluxio-presto-sandbox` in the search bar.
+Exactly one result should appear; click on its blue `Select` button to proceed.
+
+![presto_sandbox_ec2_choose_ami]({{ '/img/presto_sandbox_ec2_choose_ami.png' | relativize_url }})
+
+Note that each AMI has a unique ID in the form `ami-xxxxxxxxxxxxxxxxx`.
+This ID differs depending on which AWS region was selected.
+If you would like to launch the AMI with spot instances, click on `Community AMIs`
+and among the 2 selections, choose the one with the shorter name.
```
Comment (LuQQiu) [inline docs/en/cloud/Alluxio-Presto-AMI.md]: shorter name ... ❓ hahahahaha

File: docs/en/cloud/Alluxio-Presto-AMI.md [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+  {% collapsible Configure Security Group & Review Instance Launch %}
+
+Create a new security group named `alluxio-presto-sandbox`.
+Port 22 is already added to allow SSH access.
+Click `Add Rule` on the bottom left to add another row.
+Set the port range of the new rule to `8080` to allow access to the Presto web UI.
+Set the source to be `Anywhere`.
+
+Repeat the above steps to add another rule to open port `19999` to allow access to the Alluxio web UI.
+Click the blue `Review and Launch button on the bottom right to proceed.
```
Comment (LuQQiu) [inline docs/en/cloud/Alluxio-Presto-AMI.md]: `Review and Launch` button on the bottom right to proceed

File: docs/en/cloud/Alluxio-Presto-AMI.md [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+```
+
+You can see the current Alluxio files from the Alluxio web UI or by running `alluxio fs ls /` from within the instance.
+```console
+$ alluxio fs ls /
+drwxr-xr-x  ec2-user       ec2-user                     1       PERSISTED 07-17-2019 23:43:56:949  DIR /promotion
+drwx------                                              4       PERSISTED 07-17-2019 23:45:16:220  DIR /s3
+```
+
+The /s3 directory contains the remote data that is located in S3 (37GB)
```
Comment (LuQQiu) [inline docs/en/cloud/Alluxio-Presto-AMI.md]: /s3 -> `/s3`

File: docs/en/cloud/Alluxio-Presto-AMI.md [hunk 5]
```diff
@@ -0,0 +1,325 @@
...
+and the /promotion directory contains data from a local file (53KB) that has already been cached in Alluxio worker memory.
```
Comment (LuQQiu) [inline docs/en/cloud/Alluxio-Presto-AMI.md]: /promotion -> `/promotion`

File: docs/en/cloud/Alluxio-Presto-AMI.md [hunk 6]
```diff
@@ -0,0 +1,325 @@
...
+This setup in Alluxio allows us to run a single Presto query that utilizes data from different sources.
+  {% endcollapsible %}
+{% endaccordion %}
+
+
+## Run Queries with Presto on Alluxio
```
Comment (LuQQiu) [inline docs/en/cloud/Alluxio-Presto-AMI.md]: To be consistent, I would prefer adding an empty line under the title

File: docs/en/cloud/Alluxio-Presto-AMI.md (hunk 2 above)
Comment (Xenorith) [reply docs/en/cloud/Alluxio-Presto-AMI.md]: yes. this is the best way to describe it >.<

File: docs/en/cloud/Alluxio-Presto-AMI.md (hunk 1 above)
Comment (Xenorith) [reply docs/en/cloud/Alluxio-Presto-AMI.md]: the button text is the name of the region, not the word `region`

File: docs/en/cloud/Alluxio-Presto-AMI.md [hunk 7]
```diff
@@ -0,0 +1,328 @@
...
+  {% endcollapsible %}
+{% endaccordion %}
+
+
+## Explore Alluxio using its Web UI and CLI
+
+{% accordion explore %}
+  {% collapsible Use the Web UI and CLI %}
+We’ll use a combination of the Alluxio web UI at **http://EC2_PUBLIC_DNS:19999**
+and the [Alluxio CLI](https://docs.alluxio.io/os/user/stable/en/basic/Command-Line-Interface.html)
```
Comment (LuQQiu) [inline docs/en/cloud/Alluxio-Presto-AMI.md]: Use relative url instead?

File: docs/en/cloud/Alluxio-Presto-AMI.md [hunk 8]
```diff
@@ -0,0 +1,328 @@
...
+unlike the first query which read its data from the S3 bucket.
+Let’s run it again!
+
+**Tip:** You can hit the up arrow while in the Presto CLI to scroll through previously executed commands.
+Exit the Presto shell with `exit;`
+
+Take note of the query execution time; did you notice any difference in performance?
+If you want replicate the same result, you will need to free the data from Alluxio memory.
+After exiting the Presto shell,
+run the alluxio [free](https://docs.alluxio.io/os/user/stable/en/basic/Command-Line-Interface.html#free)
```
Comment (LuQQiu) [inline docs/en/cloud/Alluxio-Presto-AMI.md]: and also here

File: docs/en/compute/Hive.md [hunk 9]
```diff
@@ -18,7 +18,7 @@ that you can easily store Hive tables in Alluxio's tiered storage.
 * [Download and setup Hive](https://cwiki.apache.org/confluence/display/Hive/GettingStarted). If you are using Hive2.1+, 
   make sure to [run the schematool](https://cwiki.apache.org/confluence/display/Hive/GettingStarted#GettingStarted-RunningHiveServer2andBeeline.1)
   before starting Hive. `$HIVE_HOME/bin/schematool -dbType derby -initSchema`
-* Alluxio has been [set up and is running](https://docs.alluxio.io/os/user/2.0/en/deploy/Running-Alluxio-Locally.html).
+* Alluxio has been [set up and is running]({{ '/deploy/Running-Alluxio-Locally.html' | relativize_url }}).
```
Comment (LuQQiu) [inline docs/en/compute/Hive.md]: en/deploy?

File: docs/en/cloud/AWS-EMR.md [hunk 10]
```diff
@@ -83,7 +83,7 @@ The default instance type for the AMI is `r4.4xlarge`.
     For example, you can use the URL: `https://downloads.alluxio.io/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz`
     - You can also specify additional Alluxio properties as a delimited list of key-value pairs in the format `key=value`.
     For example, `alluxio.user.file.writetype.default=CACHE_THROUGH` instructs Alluxio to write files synchronously to the underlying storage system.
-    See more about [write type options](https://docs.alluxio.io/ee/user/stable/en/Architecture-DataFlow.html#data-flow-write).
+    See more about [write type options]({{ '/en/Architecture-DataFlow.html#data-flow-write' | relativize_url }}).
```
Comment (LuQQiu) [inline docs/en/cloud/AWS-EMR.md]: where is this file??

File: docs/en/cloud/Alluxio-Presto-AMI.md [hunk 11]
```diff
@@ -60,7 +60,7 @@ The AMI with the longer name is derived from the shorter one but is constrained
 
 Among the large table of instance types and their specifications, scroll down to search for the `r4.4xlarge` instance type.
 
-**Warning:** `r4.4xlarge` is provisioned with the minimum amount of resources needed to run this AMI.
+> **Warning:** `r4.4xlarge` is provisioned with the minimum amount of resources needed to run this AMI.
```
Comment (LuQQiu) [inline docs/en/cloud/Alluxio-Presto-AMI.md]: Is this needed? what it used for?

Comment (Xenorith) [reply docs/en/cloud/Alluxio-Presto-AMI.md]: styling. it shows up with a slight indentation

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please

**Pull request merged**